### PR TITLE
[BUGFIX release] Ensure Ember.Test globals setup when including ember-testing.js

### PR DESCRIPTION
**Pull request by trentmwillis**
Addressing one part of https://github.com/ember-cli/ember-cli/issues/6870.

After this, it looks like there are other errors that pop-up, but those will be handled in a separate PR.

### CODE REVIEW

**Comment by trentmwillis** [comment]
Looks like Safari bombed out on SauceLabs, due to a timeout.

**Comment by rwjblue** [comment]
restarted

File: ember-cli-build.js [hunk 1]
```diff
@@ -300,7 +300,15 @@ module.exports = function(options) {
   });
 
   emberTestingBundle = concat(emberTestingBundle, {
-    outputFile: 'ember-testing.js'
+    outputFile: 'ember-testing.js',
+    hasBootstrap: false,
+    footer: stripIndent`
+      var testing = requireModule('ember-testing');
+      Ember.Test = testing.Test;
```
Comment (rwjblue) [inline ember-cli-build.js]: Is `Ember` guaranteed to exist here? What happens when you eval `ember-testing.js` before `ember.debug.js` or `ember.prod.js`?

**Review by rwjblue — COMMENTED** [review]
Can you share the source of `dist/ember-testing.js` with this PR?

**Comment by trentmwillis** [comment]
Here's the output of [ember-testing.js](https://gist.github.com/trentmwillis/95afa3bcfda5f8d0657435346cd36444).

File: ember-cli-build.js (hunk 1 above)
Comment (trentmwillis) [inline ember-cli-build.js]: Yes, due to [these lines](https://gist.github.com/trentmwillis/95afa3bcfda5f8d0657435346cd36444#file-ember-testing-js-L19-L23). I originally had `var Ember = requireModule('ember-metal/core');` but that was flagged as a redeclaration.

Comment (rwjblue) [inline ember-cli-build.js]: Confirmed that this works as intended via shared asset in https://github.com/emberjs/ember.js/pull/15112#issuecomment-292230312

**Comment by rwjblue** [comment]
@trentmwillis - The entire build system has changed between release/beta and master, and df839ae cannot be cherry-picked.  Would you mind submitting a PR targetting beta with that commit?

**Comment by trentmwillis** [comment]
@rwjblue, sure, target beta but still tag as `BUGFIX release`?

**Comment by rwjblue** [comment]
Yes, beta and release are pretty close (the build side is basically the same), so it should still cherry-pick from beta to release.

**Comment by trentmwillis** [comment]
So turns out this needs to be ported into emberjs-build and then we'll have to bump that package on the beta/release branch here.

**Pull request merged**